tool/model.py

```diff
@@ -54,48 +62,44 @@ def __init__(self, problem, author, language, content=None, init_runnable=True):
         )
 
     def __repr__(self):
-        return "Submission{%s, by %s, in %s}" % (
-            self.problem,
-            self.author,
-            self.language,
-        )
+        return f"Submission{{{self.problem}, by {self.author}, in {self.language}}}"
```

Comment:
Ya pas un { de trop?
yen avait 1 avant et la 3

Reply:
`{{` permet d'escape les `{` dans les f-string

https://stackoverflow.com/questions/5466451/how-do-i-escape-curly-brace-characters-in-a-string-while-using-format-or

Reply:
C'est normal, dans une f-string pour faire un literal `{` if faut écrire `{{`

https://peps.python.org/pep-0498/#specification